packages/server/modules/shared/index.js

```diff
@@ -92,14 +92,20 @@ async function contextMiddleware(req, res, next) {
 
 let roles
 
+const getRoles = async () => {
```

Comment:
The issue I had with blobstorage/services/index.js goes double for this file. It will tend to collect all kinds of random stuff and have a ton of different concerns. Moving stuff to a common/shared folder makes sense to me, but the shared code should still be organized according to concern and no single module should have too many concerns.

I realize that this file already existed beforehand, so this isn't necessarily about your addition, but a comment about the situation generally

Reply:
here i agree 100%. This will get cleaned up with the authz/middleware rework